src/main/java/com/gitblit/git/PatchsetReceivePack.java

```diff
@@ -1268,7 +1268,28 @@ private void updateReflog(RefUpdate ru) {
 	public MergeStatus merge(TicketModel ticket) {
 		PersonIdent committer = new PersonIdent(user.getDisplayName(), StringUtils.isEmpty(user.emailAddress) ? (user.username + "@gitblit") : user.emailAddress);
 		Patchset patchset = ticket.getCurrentPatchset();
-		String message = MessageFormat.format("Merged #{0,number,0} \"{1}\"", ticket.number, ticket.title);
+		StringBuilder messageBuilder = new StringBuilder();
+		messageBuilder.append(MessageFormat.format("Merged #{0,number,0} \"{1}\"", ticket.number, ticket.title));
+
+		// Add Signed-off-by tags to commit message footer if there are reviewers on this patchset
+		// and the setting is enabled on this repo.
+		if (repository.writeSignoffCommit) {
+			messageBuilder.append("\n\n");
+			for (Change change : ticket.getReviews(patchset)) {
+				UserModel ruser = gitblit.getUserModel(change.author);
+				messageBuilder.append(MessageFormat.format(

```

Comment:
This looks like it would add a `sign-off` for someone who gave a negative review but the total score exceeded the minimum requirement.  Am I reading that wrong?